pyrasterframes/src/main/python/pyrasterframes/rf_types.py

```diff
@@ -390,6 +390,11 @@ def __init__(self, cells, cell_type=None):
                 # if the value in the array is `nd_value`, it is masked as nodata
                 self.cells = np.ma.masked_equal(self.cells, nd_value)
 
+        # is it a buffer tile? crop it on extraction to preserve the tile behavior
```

Comment:
Not sure this is the right thing to do, but don't want to hold up the PR for this. Back to the "everything's a buffered tile" perspective, I'd want to explore supporting buffered tile semantics on the Python mirror type.

Reply:
@metasim yes, for now it is done to preserve the old behavior and not to break anything.